app/views/adoption_applications/new.html.erb

```diff
@@ -0,0 +1,120 @@
+<h1> Adoption Application </h1>
+
+<% if @adoption_application.errors.any? %>
+  <div id="error_explanation">
+    <h2><%= pluralize(@adoption_application.errors.count, "error") %> prevented your application from being submitted:</h2>
+ 
+    <ul>
+    <% @adoption_application.errors.full_messages.each do |message| %>
+      <li><%= message %></li>
+    <% end %>
+    </ul>
+  </div>
+<% end %>
+
+<div class="form_questions">
+  <%= form_for @adoption_application do |f| %>
+    <%= f.label "name:" %>
```

Comment:
You'll want the label to match you text field, so this and all the labels should be the same symbol as the line below i.e.

```html.erb
<%= f.label :name %>
```

If you don't want to have the same label as the column name you can change it using a translation: http://apidock.com/rails/ActionView/Helpers/FormHelper/label

If you want to just add a colon I think you can do that with css.